test_data/utilities_testdata.py

```diff
@@ -0,0 +1,312 @@
+test_record_type = 'digital_objects'
```

Comment:
This all is a good stop-gap measure for getting some reusable test data into the repo, but will obviously get pretty unwieldy over time.  As time permits, coming up with a plan for test fixtures/test data management is probably a good next big question to handle.  But this takes us in the right general direction.